Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM golang:alpine AS builder
+FROM golang:1.22-alpine3.20 AS builder
 
 RUN apk add --no-cache --update git gcc rust
 
 COPY . /src
 WORKDIR /src
 
-RUN CGO_ENABLED=0 go build -a -ldflags "-linkmode external -extldflags -static" -o /usr/local/bin/supervisord github.com/ochinchina/supervisord
+RUN CGO_ENABLED=1 go build -a -ldflags "-linkmode external -extldflags -static" -o /usr/local/bin/supervisord github.com/cyralinc/supervisord
```

Comment:
Why do you want to set CGO_ENABLED to 1 here?

Reply:
I had to change it to build in MacOS and forgot to rollback the change.

Reply:
And yes, I tested and it runs fine.

Reply:
Addressed in [eee8ff8](https://github.com/cyralinc/supervisord/pull/2/commits/eee8ff890b2c60d320dd3216e33ed06aeff2e4a6).